pipelines/azure-pipelines-nuget.yml

```diff
@@ -0,0 +1,388 @@
+        xcopy /E $(ArtifactsFolder)\LINUXchk               package_linux\build\native\LINUXchk\
+        xcopy /E $(ArtifactsFolder)\LINUXchk-pack          package_linux\build\native\LINUXchk-pack\
+      displayName: 'Prepare Linux'
+
+    - task: NuGetCommand@2
+      displayName: 'NuGet pack Linux'
+      inputs:
+        command: pack
+        packagesToPack: 'package_linux\Package.nuspec'
+        packDestination: '$(NuGetsFolder)\linux'
+
+    # MacOS
+    # For macOS we first sign the libraries
+    - script: |
+        mkdir ToSign
+        mkdir ToSign\MACOS
+        mkdir ToSign\MACOS-nobundle
+        mkdir ToSign\MACOS-nobundle-Universal
+        mkdir ToSign\MACOS-pack
+        mkdir ToSign\MACOS-pack-Universal
+        mkdir ToSign\MACOS-Universal
+        mkdir ToSign\MACOSarm64
+        mkdir ToSign\MACOSarm64-nobundle
+        mkdir ToSign\MACOSarm64-pack
+        mkdir ToSign\MACOSarm64chk
+        mkdir ToSign\MACOSarm64chk-pack
+        mkdir ToSign\MACOSchk
+        mkdir ToSign\MACOSchk-pack
+
+        copy $(ArtifactsFolder)\MACOS\lib\libmsix.dylib                    ToSign\MACOS\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOS-nobundle\lib\libmsix.dylib           ToSign\MACOS-nobundle\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOS-nobundle-Universal\lib\libmsix.dylib ToSign\MACOS-nobundle-Universal\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOS-pack\lib\libmsix.dylib               ToSign\MACOS-pack\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOS-pack-Universal\lib\libmsix.dylib     ToSign\MACOS-pack-Universal\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOS-Universal\lib\libmsix.dylib          ToSign\MACOS-Universal\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOSarm64\lib\libmsix.dylib               ToSign\MACOSarm64\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOSarm64-nobundle\lib\libmsix.dylib      ToSign\MACOSarm64-nobundle\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOSarm64-pack\lib\libmsix.dylib          ToSign\MACOSarm64-pack\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOSarm64chk\lib\libmsix.dylib            ToSign\MACOSarm64chk\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOSarm64chk-pack\lib\libmsix.dylib       ToSign\MACOSarm64chk-pack\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOSchk\lib\libmsix.dylib                 ToSign\MACOSchk\libmsix.dylib
+        copy $(ArtifactsFolder)\MACOSchk-pack\lib\libmsix.dylib            ToSign\MACOSchk-pack\libmsix.dylib
+      displayName: 'Prepare to Sign macOS'
+
+    - task: ArchiveFiles@2
+      displayName: 'Archive files to sign'
+      inputs:
+        rootFolderOrFile: ToSign
+        includeRootFolder: false
+        archiveFile: 'Signed\artifacts.zip'
+
+    - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+      displayName: 'ESRP CodeSigning macOS dylibs'
+      inputs:
+        ConnectedServiceName: 'ESRP Connection MsixSDK 2'

```

Comment:
>ESRP Connection MsixSDK 2 [](http://example.com/codeflow?start=31&length=25)

We should make this a build parameter, so when we have to renew it we don't have to create new PR.

Reply:
Why would we have to create a new PR? I'd think we could just edit the existing connection.